torch/_torch_docs.py

```diff
@@ -8374,38 +8379,34 @@ def merge_dicts(*dicts):
                tensors for `U` and `Vh`, whereas :func:`torch.linalg.svd` returns
                empty tensors.
 
-.. note:: The singular values are returned in descending order. If :attr:`input` is a batch of matrices,
-          then the singular values of each matrix in the batch are returned in descending order.
-
 .. note:: The `S` tensor can only be used to compute gradients if :attr:`compute_uv` is `True`.
 
 .. note:: When :attr:`some` is `False`, the gradients on `U[..., :, min(m, n):]`
           and `V[..., :, min(m, n):]` will be ignored in the backward pass, as those vectors
           can be arbitrary bases of the corresponding subspaces.
 
-.. note:: The implementation of :func:`torch.linalg.svd` on CPU uses LAPACK's routine `?gesdd`
-          (a divide-and-conquer algorithm) instead of `?gesvd` for speed. Analogously,
+.. note:: The implementation of :func:`torch.linalg.svd` on CPU uses LAPACK's routine `gesdd`
```

Comment:
but this is torch.svd, not torch.linalg.svd

I thought we were not bothering with torch.svd changes for now?

Reply:
Sorry, I just revered the file.